package.json

```diff
@@ -162,9 +174,10 @@
     "simple-git": "^3.27.0",
     "stream-browserify": "^3.0.0",
     "style-loader": "^4.0.0",
-    "ts-jest": "^29.3.4",
-    "typescript": "^5.8.3",
-    "webpack": "^5.99.8",
+    "typescript": "^5.7.3",
+    "vite": "^4.4.1",
```

Comment:
`vitest` requires importing `vite` too? Because we already use `esbuild` (and I have some plans to get rid of webpack for the production builds too).